### PR TITLE
Enables and facilitates wide screen spessmen

### PR DESCRIPTION
**Pull request by d3athrow**
I'm sure i fucked up somewhere or forgot something
:cl:
 * rscadd: Widescreen spacemen

### CODE REVIEW

**Comment by Exxion** [comment]
New powergaming strat: Ambush from above
Also, how does this interact with `client.dir`? Not that anything else in the codebase plays nice with it either

**Comment by d3athrow** [comment]
`Also, how does this interact with client.dir? Not that anything else in the codebase plays nice with it either`
It is not accounted for, does anything even set client.dir other than admin abuse variable editing mid round?

**Comment by Exxion** [comment]
Mindbreaker/Spiritbreaker

**Comment by DeityLink** [comment]
I was also pondering whether the Confusion rune in Cult 3.0 could mess with it.

**Comment by d3athrow** [comment]
I just searched for client.dir in the codebase and got no hits 🤔

**Comment by Exxion** [comment]
https://github.com/vgstation-coders/vgstation13/blob/80f35b7d06c1dab7bb9ff18777c95874b968addd/code/modules/flufftext/Hallucination.dm#L346

File: code/game/objects/items/robot/robot_items/robot_binoculars.dm [hunk 1]
```diff
@@ -19,12 +19,18 @@
...
+			if(!widescreen)
+				view[1] = ((view[1] * 2) + 1)
+				view[2] = ((view[2] * 2) + 1)
+
 			if(zoom && R.is_component_functioning("camera"))
 				event_key = R.on_moved.Add(src, "mob_moved")
 				R.visible_message("[R]'s camera lens focuses loudly.","Your camera lens focuses loudly.")
 				R.regenerate_icons()
-				C.changeView(C.view + 4)
+				C.changeView("[view[1]+8]x[view[2]+8]")
```
Comment (jknpj) [inline code/game/objects/items/robot/robot_items/robot_binoculars.dm:32]: image

Something really weird going on

**Comment by d3athrow** [comment]
lmao didn't know that label would do that, i just wanted to meme

**Review by d3athrow — COMMENTED** [review]
@jknpj Should be fixed with https://github.com/vgstation-coders/vgstation13/pull/19693/files#diff-2aec593799a15c879d8f95f7276b689aR1802 commented out.

**Comment by Durgen** [comment]
does anyone else think this is a bad idea?

**Comment by DeityLink** [comment]
I'd like to see it for myself at least once.

**Comment by jknpj** [comment]
Looks good, but since i've only been compiling test tiny i don't know how the splash screens look

**Comment by Exxion** [comment]
@Durgen Yes, but I too would like to see it anyway

**Comment by d3athrow** [comment]
What is the worst that can happen?

**Comment by Exxion** [comment]
Pretty much everything in the game expects a square view, and being able to see farther along one axis just because screens are wider than they are tall is weird
But I didn't say we shouldn't try it

**Comment by Durgen** [comment]
@d3athrow giving players with widescreen monitors a significant advantage over those without them and changing the timeless square aesthetic

**Comment by DeityLink** [comment]
Basically what @Durgen said. I'd like to see it once just to be sure it's a bad idea but I'd like it reverted eventually. Plus that means less space for the chat.

**Comment by d3athrow** [comment]
@Durgen Who doesn't have a 16:9/16:10 monitor by now?
@DeityLink If you want to see what its like just test it locally, don't +1 a PR you know you already want to revert later.

**Comment by Durgen** [comment]
@d3athrow people who have something like 21:9 or third world frens with 4:3. Regardless, it would be a major change to the aesthetic and gameplay of spess.

**Comment by Hugh-G-Rection** [comment]
Tie the ability to an in-game item and this would be a great addition. Captain starts with it. With a spare one on his desk. For reasons.

**Comment by CthulhuOnIce** [comment]
This seems pretty interesting tbh, widescreen has worked on other servers. 
Maybe make a verb/preference that toggles between the two modes. (Widescreen and non-widescreen)

**Comment by Dacendeth** [comment]
wont this look horrible with blindness overlays, that stuff already messes up when you have the eyesight genetic power active?

**Comment by DamianX** [comment]
I resized my window to make it look like this:
image

Is this how I'm supposed to be using it? Why is it not the default, then?

**Comment by DamianX** [comment]
Full-screen overlays need to be adjusted
image

**Comment by DamianX** [comment]
The main screen now shows space to the side:
image

xenos have this thing in the middle of the screen:
image

tooltips appear in a completely wrong position:
image

**Review by Intigracy — CHANGES_REQUESTED** [review]
Conflicts as well as the above images.

**Pull request closed without merging**